src/js/core/widget/core/VirtualListview.js

```diff
@@ -509,6 +512,11 @@
 				self._currentIndex = index;
 			}
 
+			function getFirstElementChild(element) {
+				return (element.firstElementChild.tagName !== "CANVAS") ?
```

Comment:
Please consider changing this condition to opposite logic and return first <li> element.

Reply:
Done